examples/api/README.txt

```diff
@@ -1,39 +1,18 @@
-matplotlib API
+.. _api_examples:
+
+Matplotlib API
 ==============
 
-These examples use the matplotlib api rather than the pylab/pyplot
+These examples use the Matplotlib api rather than the pylab/pyplot
 procedural state machine.  For robust, production level scripts, or
 for applications or web application servers, we recommend you use the
-matplotlib API directly as it gives you the maximum control over your
+Matplotlib API directly as it gives you the maximum control over your
 figures, axes and plottng commands.
 
 The example agg_oo.py is the simplest example of using the Agg backend
 which is readily ported to other output formats.  This example is a
 good starting point if your are a web application developer.  Many of
-the other examples in this directory use matplotlib.pyplot just to
+the other examples in this directory use ``matplotlib.pyplot`` just to
 create the figure and show calls, and use the API for everything else.
 This is a good solution for production quality scripts.  For full
 fledged GUI applications, see the user_interfaces examples.
-
-Example style guide
```

Comment:
This is not the correct place to document example style guide. We have a MEP for this in addition of contributors guidelines on how to write documentation. This section is quite out of date.

In addition, the READMEs are now being rendered in the gallery: they thus should contain documentation relevant to the end user on this specific section of the gallery instead of a mixture of user documentation and contribution guidelines.

Does this make sense?